src/config.h.in

```diff
@@ -74,38 +82,44 @@
 #define DSS_PROC 1
 #endif
 
+#cmakedefine HAVE_KILL
+#cmakedefine HAVE_FORK
+#cmakedefine HAVE_WAIT
+
 #define RNG_A	6364136223846793005ull
 #define RNG_C	1ull
 
-#ifdef __APPLE__ 
-#define _POSIX_C_SOURCE 200112L
-#endif 
-
 #if (defined(_WIN32) && !defined(_POSIX)) || defined(__CYGWIN__)
 #define PATH_SEP '\\'
 #else
 #define PATH_SEP '/'
 #endif
 
-#if (defined(_WIN32) && !defined(_POSIX)) || defined(__CYGWIN__)
-#define pid_t int
-#define SET_HANDLER(proc) signal(SIGINT, proc)
-#define KILL(pid) TerminateProcess(OpenProcess(PROCESS_TERMINATE,FALSE,pid),3)
-#if (defined (__WATCOMC__))
-#define SPAWN()   spawnv(P_NOWAIT, spawn_args[0], spawn_args)
-#define WAIT(res, pid) cwait(res, pid, WAIT_CHILD)
-#else /* MSVC (?) */
-#define SPAWN()   _spawnv(_P_NOWAIT, spawn_args[0], spawn_args)
-#define WAIT(res, pid) _cwait(res, pid, _WAIT_CHILD)
-#define getpid          _getpid
-#endif /* WATCOMC */
-#else
-#define KILL(pid) kill(SIGUSR1, pid)
-#define SET_HANDLER(proc) signal(SIGUSR1, proc)
-#define SPAWN   fork
-#define WAIT(res, pid) wait(res)
+#if (!defined(HAVE_UNISTD_H) && !defined(HAVE_PID_T_IN_SYS_TYPES_H))
+	typedef int pid_t;
 #endif
 
+// These macros aren't being used in code currently and parallel load is allowed 
+// only for POSIX-compatible systems (with fork, kill and wait).
+//#if (defined(HAVE_UNISTD_H) && defined(HAVE_SYS_WAIT_H))
+//#define KILL(pid) kill(SIGUSR1, pid)
+//#define SET_HANDLER(proc) signal(SIGUSR1, proc)
+//#define SPAWN   fork
+//#define WAIT(res, pid) wait(res)
+//#else
+//#define SET_HANDLER(proc) signal(SIGINT, proc)
+//#define KILL(pid) TerminateProcess(OpenProcess(PROCESS_TERMINATE,FALSE,pid),3)
+//
+//#if (defined (__WATCOMC__))
+//#define SPAWN()   spawnv(P_NOWAIT, spawn_args[0], spawn_args)
+//#define WAIT(res, pid) cwait(res, pid, WAIT_CHILD)
+//#else /* MSVC (?) */
+//#define SPAWN()   _spawnv(_P_NOWAIT, spawn_args[0], spawn_args)
+//#define WAIT(res, pid) _cwait(res, pid, _WAIT_CHILD)
+//#define getpid          _getpid
+//#endif /* (defined (__WATCOMC__)) */
+//#endif /* (defined(HAVE_UNISTD_H) && defined(HAVE_SYS_WAIT_H)) */
+
```

Comment:
I'm not super-happy about using `//` comments, since we don't have them anywhere else in the code. But... fine, I guess. Can you add `//-----` and `//----` delimiters and say "The following commented-out macros appeared in the original ssb-dbgen code, but were not being used, with parallel load only made possible for POSIX-compatible systems (using fork, kill and wait)."